pbm/storage/webdav/webdav.go

```diff
@@ -0,0 +1,125 @@
+package webdav
+
+import (
+	"github.com/percona/percona-backup-mongodb/pbm/log"
+	"github.com/percona/percona-backup-mongodb/pbm/storage"
+	"github.com/pkg/errors"
+	"github.com/studio-b12/gowebdav"
+	"io"
+	"os"
+	"path"
+	"strings"
+)
+
+type Conf struct {
+	ServerURL   string      `bson:"serverUrl,omitempty" json:"serverUrl" yaml:"serverUrl,omitempty"`
+	Prefix      string      `bson:"prefix,omitempty" json:"prefix,omitempty" yaml:"prefix,omitempty"`
+	Credentials Credentials `bson:"credentials" json:"-" yaml:"credentials"`
```

Comment:
I've copied this over from `s3.go` and `azure.go`.
But I'm wondering, is there a reason why it's set to `-` only for `json`?

Reply:
Not sure why. I guess it to not be shown on debug output. The `bson:` is required to be stored in MongoDB.

Reply:
👍